dadi/lib/composer/index.js

```diff
@@ -89,6 +83,13 @@ Composer.prototype.composeOne = function (doc, callback) {
             }
           }
 
+          // Are we composing a media document? If so, we need to format it
```

Comment:
@jimlambie I don't like this. It feels like we should have the ability to have built-in hooks (as in not actual files on `workspace/hooks`) that would process documents. In this case, a hook would check whether a document is a media asset and, if so, remove some of the properties and change the relative path to a full URL.